Test/SilacTest.cs

```diff
@@ -212,8 +212,11 @@ public static void TestSilacQuantification()
             //test proteins
             string[] output = File.ReadAllLines(TestContext.CurrentContext.TestDirectory + @"/TestSilac/AllQuantifiedProteinGroups.tsv");
             Assert.AreEqual(output.Length, 2);
-            Assert.IsTrue(output[0].Contains("Intensity_silac\tIntensity_silac(K+8.014)")); //test that two files were made
-            Assert.IsTrue(output[1].Contains("1237436.8670764582\t618718.4335382291")); //test the heavy intensity is half that of the light (per the raw file)
+            Assert.IsTrue(output[0].Contains("Intensity_silac\tIntensity_silacPart2\tIntensity_silac(K+8.014)\tIntensity_silacPart2(K+8.014)")); //test that two files were made
```

Comment:
Why are there four headings now? What is Part2?

Reply:
This part is a bit confusing to me, but hopefully you can clarify.. the unit test seems to me to be 2 spectra files which are identical, and 2 labels. So there should be 4 columns in the output right? 2 light and 2 heavy columns? 1 light/heavy for each spectra file?

Reply:
Yep, you're right. Looks like I was lazy in my original coding for this test.